test/suite/test_txn02.py

```diff
@@ -93,6 +93,13 @@ class test_txn02(wttest.WiredTigerTestCase, suite_subprocess):
     checklog_calls = 100 if wttest.islongtest() else 2
     checklog_mod = (len(scenarios) / checklog_calls + 1)
 
+    _debug = False
+    def debug(self, msg):
+        if not self._debug:
+            return
+
```

Comment:
I'm okay with this function. I'd suggest removing the blank line here so the `print(msg)` isn't just floating on its own.